Jenkinsfile.migration

```diff
@@ -11,65 +11,120 @@
  *
  * Removal or modification of this copyright notice is prohibited.
  */
+
 pipeline {
-	agent { node { label "node-07" } }
+	agent { node { label "core-migration" } }
 	stages {
 		stage("Prepare workspace") {
 			steps {
-				deleteDir()
-				dir(params.MIGRATE_FROM) {
-					git url: "https://github.com/LiskHQ/lisk.git", branch: params.MIGRATE_FROM
-				}
-				dir(params.MIGRATE_TO) {
-					git url: "https://github.com/LiskHQ/lisk.git", branch: params.MIGRATE_TO
+				library identifier: 'lisk-jenkins@master', retriever: modernSCM(
+  					[$class: 'GitSCMSource',
+   					 remote: 'https://github.com/LiskHQ/lisk-jenkins.git'])
+			}
+		}
+		stage("Create binary releases") {
+			parallel {
+				stage('Create source release') {
+					steps {
+						script {
+							def release = liskBuild(branch: params.MIGRATE_FROM, network: 'test', useTestConfig: true)
+							env.SRC_IS_LATEST = release.latestVersion
+							env.SRC_VERSION =  release.version
+							env.SRC_RELEASE =  release.nonVersionedFile
+						}
+
+					}
 				}
-				sh """
-				json_files_copy() {
-					case \$1 in
-						0* | master)
-							cp test/config.json .
-							cp test/genesisBlock.json .
-							;;
-						*)
-							cp test/data/config.json .
-							cp test/data/genesis_block.json .
-							;;
-					esac
+				stage('Create destination release') {
+					steps {
+						script {
+							def release = liskBuild(branch: params.MIGRATE_TO, network: 'test', useTestConfig: true)
+							env.DEST_IS_LATEST = release.latestVersion
+							env.DEST_VERSION =  release.version
+							env.DEST_RELEASE =  release.nonVersionedFile
+						}
+					}
 				}
+			}
+		}
 
-				dropdb lisk_test || true
-				createdb lisk_test
-				cd "${params.MIGRATE_FROM}"
-				json_files_copy "${params.MIGRATE_FROM}"
-				npm install
-				cd "../${params.MIGRATE_TO}"
-				json_files_copy "${params.MIGRATE_TO}"
-				npm install
-				sudo service postgresql restart
```

Comment:
Where and when is DB migration happening now?

Reply:
Inside the core itself. When we start the node with `bash lisk.sh start` it will start the node and the DB migration will happen. In current structure `lisk_bridge.sh` triggers the `installLisk.sh upgrade` and that triggers the `lisk.sh start`